### ISSUE TITLE
consider using outline buttons throughout the app

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#1001]
A quick check of the contrast of buttons shows `btn-danger` doesn't have high enough contrast (we use this to delete sessions in the session card) though `btn-primary` seems ok.

In any case, it was brought up that maybe we should use `btn-outline` instead so this issue is to explore that.

https://getbootstrap.com/docs/4.0/components/buttons/#outline-buttons

**Comment by johrstrom** [comment]
Additionally - during the bootstrap 5 upgrade we found that we override the color red for btn-danger, so maybe it's our fault entirely.

**Comment by moffatsadeghi** [comment]
btn outline could be a solution for this, but the scope is probably not a good first issue due to the tracking down of all the pieces.  Removing from 4.1

**Comment by Bubballoo3** [comment]
Will mark as a wontfix due to the large impact of using outlines instead of modifying our existing color defaults. I've created #5169 to check the contrast
